docs/API_action_commands.md

```diff
@@ -0,0 +1,31 @@
+# Action API supported commands
+
+## Supported actions
+These actions are currently supported using the [Action API](API.md#ActionAPI)
+
+None at this time.
+
+----
+## Actions under development
+These actions are under active development
+
+* deploy site  
+Triggers the initial deployment of a site, using the staged design and secrets
+
+* update site  
```

Comment:
Let's discuss more. I'm thinking for the near term, we're not going to have time to distinguish separate flows, but depending on the complexity it might be possible.